Header header
float32 total_loss
float32 loss_align
float32 loss_couple
float32 loss_smooth